iTechArt.Hotels.Database/Hotels.sql

```diff
@@ -0,0 +1,8 @@
+﻿CREATE TABLE [dbo].[Hotels] (
+    [HotelId] INT IDENTITY (1, 1) NOT NULL,
+    [Name] VARCHAR(60) NOT NULL,
+	[Country] VARCHAR(60) NOT NULL,
+	[Sity] VARCHAR(60) NOT NULL,
```

Comment:
Does any address include city?

Reply:
No, for me city is not a part of the address